CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - On Windows, fix `CursorMoved(0, 0)` getting dispatched on window focus.
 - On macOS, fix command key event left and right reverse.
 - On FreeBSD, NetBSD, and OpenBSD, fix build of X11 backend.
+- X11 - Dpi scaling factor behavior changed. First, winit tries to read it from "Xft.dpi" XResource, and uses DPI calculation from xrandr dimensions as fallback behavior.
```

Comment:
Nit pick: can you make this change log entry like the rest? 
"On X11, ..." ->"X11 - ..."